src/extension.ts

```diff
@@ -134,6 +134,43 @@ export async function activate(context: vscode.ExtensionContext): Promise<void>
 		// add commands to create config-map and secret objects from .properties files
 		configmapsandsecrets.registerCommands();
 		
+		// create the integration view action -- open operator log
+		vscode.commands.registerCommand('camelk.integrations.openOperatorLog', async () => {
+			setStatusLineMessageAndShow(`Retrieving log for Apache Camel K Operator...`);
+			await kubectlutils.getNamedPodsFromKubectl(`camel-k-operator`).then( async (podNames) => {
+				await handleLogViaKubectlCli(podNames[0]).catch((error) => {
```

Comment:
why is it taking the first one? What happens if there are several?

Reply:
there should only be one for a given namespace

Reply:
but I am asking the runtime guys to verify

Reply:
Seems based on info from the runtime folks that there will be at most 1, but may be none.